lib/lightdata.h

```diff
@@ -12,17 +12,58 @@ class lightdata {
   unsigned char fine = 0;
   unsigned char tdc = 0;
 
-  int chip() const { return index / 32; };
-  int eoch() const { return index % 64; };
-  int cindex() const { return tdc + 4 * index; };
-  
   lightdata(unsigned char _device, unsigned char _index, unsigned char _coarse, unsigned char _fine, unsigned char _tdc) :
     device(_device),
     index(_index),
     coarse(_coarse),
     fine(_fine),
     tdc(_tdc) { };
-    
+
+  int chip() const { return index / 32; };
+  int eoch() const { return index % 64; };
+  int cindex() const { return tdc + 4 * index; };
+  float time() const;
+
+  /** calibration **/
+
```

Comment:
// IF, CUT, OFF from fine tune calibration [device (16 ?) ][calibration_index]

Reply:
comment not clear